src/main/kotlin/model/algos/SPAFordBellman.kt

```diff
@@ -6,7 +6,7 @@ import model.graph.edges.Edge
 typealias Path<V> = List<Edge<V>>
 typealias Paths<V> = Map<V, Path<V>>
 
-object FordBellman {
+object SPAFordBellman {
```

Comment:
Зачем добавлять какие-то сложные аббревиатуры, которые ещё приходится расшифровывать и каждый раз вводить, когда надо вызвать функцию алгоритма?
```suggestion
object FordBellman {
```

Reply:
Да зато сразу понятно что это за алгоритм.
Лан, потом отрефакторим это